### ISSUE TITLE
[feature request] add vr/isearch-query-replace-regexp as anzu.

### ISSUE THREAD

**Issue by zw963** [benma/visual-regexp.el#65]
https://github.com/emacsorphanage/anzu#anzu-isearch-query-replace-regexp

`vr/isearch-query-replace-regexp` should only be used on isearch mode after use `vr/isearch` get expected result,
and don't exit iserach, run this command directly replace on exists search.

Thank you.

**Comment by benma** [comment]
Sorry, I don't understand the request. Could you rephrase it? Thanks.

**Comment by zw963** [comment]
Okay let us reproduce this issue.

### 1.  let us open `visual-regexp.el`

### 2. let us search `vr/` use `vr/isearch-forward`, it works!

### 3.  now, i want to replace all matched `vr/` one by one to `vr-`, now, what i expected is there exist a `vr/isearch-query-replace-regexp`, when i execute it, it can use exists `vr/` search result directly, and let user input the replacement directly.

For now, there is no a equivalent i can use as above, right? i use exists command, we have to input the `vr/` again even current iserach mode is active.
